### PR TITLE
Updating lab api package version and ESRP config

### PR DESCRIPTION
**Pull request by trwalke**
# Updating lab api package version and ESRP config
Updating lab api package version to resolve cert lookup issues when running on a machine with an expired version of the desired cert.

- [x] You've read the [Contributor Guide](https://github.com/AzureAD/microsoft-identity-web/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/AzureAD/microsoft-identity-web/blob/master/CODE_OF_CONDUCT.md).
- [x] You've included unit or integration tests for your change, where applicable.
- [x] You've included inline docs for your change, where applicable.
- [ ] There's an open issue for the PR that you are making. If you'd like to propose a new feature or change, please open an issue to discuss the change or find an existing issue.

Summary of the changes (Less than 80 chars)

## Description

{Detail}

Fixes #{bug number} (in this specific format)

### CODE REVIEW

**Review by jmprieur — APPROVED** [review]
LGTM
Thanks @trwalke 

Where is the PR for the fix in the lab project?

**Comment by trwalke** [comment]
> LGTM Thanks @trwalke
> 
> Where is the PR for the fix in the lab project?

https://identitydivision.visualstudio.com/Engineering/_git/Microsoft.Identity.Lab.Api/pullrequest/9418

**Review by westin-m — APPROVED** [review]

File: build/template-pack-and-sign-nuget.yaml [hunk 1]
```diff
@@ -27,8 +27,8 @@ steps:
            "parameterValue": "Microsoft.Identity.Web"
          },
          {
-           "parameterName": "Microsoft",
-           "parameterValue": "http://www.microsoft.com"
+           "parameterName": "OpusInfo",
+           "parameterValue": "https://www.microsoft.com"
```
Comment (jennyf19) [inline build/template-pack-and-sign-nuget.yaml:31]: Let's use http instead. thanks

**Comment by jennyf19** [comment]
thanks @trwalke

**Pull request merged**